<?xml version="1.0"?>
<launch>
    <!-- Load parameters -->
    <rosparam command="load" file="$(find navigation)/param/ego_planner.yaml" />

    <!-- Start ego planner node -->
    <node pkg="navigation" type="ego_planner" name="ego_planner" output="screen">
        <remap from="~cloud" to="/airsim_node/drone_1/pointcloud/filtered"/>
        <remap from="~goal" to="/move_base_simple/goal"/>
        <remap from="~pose" to="/airsim_node/drone_1/pose"/>
    </node>

    <!-- Start rviz -->
    <!-- <node pkg="rviz" type="rviz" name="rviz" args="-d $(find navigation)/rviz/drone_navigation.rviz" /> -->
</launch> 